WORKSPACE

```diff
@@ -18,7 +18,7 @@ deps_gen()
 load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
-NODE_VERSION = "22.15.1"
+NODE_VERSION = "22.18.0"
```

Comment:
Note that node 22.18.0 was added to rules_nodejs [last week](https://github.com/bazel-contrib/rules_nodejs/commit/356c5ff7e8804164bf9d13c3e9536579704b3b0f), but it is only available in v6.5.0 while workerd is still on 6.4.0 – not sure if it matters (I think we won't be verifying the sha checksum if the version is not registered in rules_nodejs?) but this would be a great time to also update rules_nodejs.

Reply:
This PR also updates rules_nodejs